tools/install.py

```diff
@@ -76,7 +77,7 @@ def run(ip_addr, hostname, config, scripts, user, toolchain, **kwargs):
 
     # Get list of config files.
     config_files = [os.path.relpath(c, build_dir) for c in b.cmake_cache['NUCLEAR_MODULE_DATA_FILES']]
-
+    
```

Comment:
Your sublime should be removing trailing white space? Are you editing in sublime with the project file open?